app/model/persisted/assessor/AssessorAllocation.scala

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
I think we should move this new file to something like  app/model/persisted/assessorallocation/AssessorAllocation.scala

or even 
app/model/persisted/AssessorAllocation.scala

I view the package as a way to say which is the main domain or collection but at this moment Assessor and AssessorAllocation are somehow separate.

To be honest, not sure why we created assessor package, there is only one file and the file is quite simple.